commands/root.go

```diff
@@ -94,6 +93,8 @@ func Execute() error {
 	Root.PersistentFlags().StringVarP(&util.WorkDir, "work-dir", "w", "~/.skywalking-infra-e2e", "the working directory for skywalking-infra-e2e")
 	Root.PersistentFlags().StringVarP(&util.LogDir, "log-dir", "l", "~/.skywalking-infra-e2e/logs", "the container logs directory for environment")
 	Root.PersistentFlags().StringVarP(&util.CfgFile, "config", "c", constant.E2EDefaultFile, "the config file")
+	Root.PersistentFlags().BoolVarP(&util.BatchOutput, "batchOutput", "b", true,
```

Comment:
```suggestion
	Root.PersistentFlags().BoolVarP(&util.BatchMode, "batch-mode", "B", true,
```

1. Rename to "batch mode" so we can disable all (future) reactive operations, not just output
2. Change naming style to kebab-case  so it's consistent with other flags. 
3. Use `-B` as the shorthand flag as it's common in other softwares.

Reply:
Make sense to me.